streamlit/streamlit_app.py

```diff
@@ -0,0 +1,289 @@
+"""An example of Streamlit leveraging Wise pizza."""
+
+import streamlit as st
+import os, sys
+import pandas as pd
+
+import warnings
+warnings.filterwarnings("ignore")
+
+root_path = os.path.realpath('../..')
+
+# this assumes that all of the following files are checked in the same directory
+sys.path.append(os.path.join(root_path,"wise-pizza"))
+
+# create data-related directories
+data_dir = os.path.realpath(os.path.join(root_path, 'wise-pizza/data'))
+if not os.path.isdir(data_dir):
+    os.mkdir(data_dir)
```

Comment:
would be nice to be able to launch the app from any folder (didn't work for me initially and I had to change the root_path)
You could add the streamlit folder to the wise-pizza folder and make the relevant changes to achieve this

Reply:
Must have been solved, please try this again!